Source/WebCore/workers/service/server/SWRegistrationDatabase.cpp

```diff
@@ -433,7 +433,7 @@ std::optional<Vector<ServiceWorkerScripts>> SWRegistrationDatabase::updateRegist
 
     for (auto& registration : registrationsToDelete) {
         auto statement = cachedStatement(StatementType::DeleteRecord);
-        if (!statement || !statement->bindText(1, registration.toDatabaseKey()) || statement->step() != SQLITE_DONE) {
+        if (!statement || statement->bindText(1, registration.toDatabaseKey()) != SQLITE_OK || statement->step() != SQLITE_DONE) {
```

Comment:
I don't understand how `!statement->bindText(...)` gives a different answer than `statement->bindText(...) != SQLITE_OK`, since `SQLITE_OK` is defined as `0`. Is it possible that we want to be checking for multiple things like maybe `SQLITE_DONE` or something?

Reply:
On the other hand, all other instances of `bindText(...)` use this specific equality check, so this change is good if only as a way of making the code more consistent.